README.md

```diff
@@ -314,6 +314,10 @@ The following resources _CAN_ be created:
 | redis\_snapshot\_window | Redis snapshot window | string | `"00:00-05:00"` | no |
 | redis\_subnet\_tag\_filter | The Map to filter the subnets of the VPC where the Redis component of the Microservice resides | map | `{}` | no |
 | redis\_transit\_encryption\_enabled | Redis encrypt transit TLS | string | `"false"` | no |
+| s3\_enabled | S3 bucket creation and iam policy creation enabled | bool | `"true"` | no |
+| s3\_force\_destroy | S3 Force destroy | bool | `"true"` | no |
+| s3\_identifier\_override | S3 naming overrride | string | `""` | no |
```

Comment:
So if this is an overwrite, what is the default?

Reply:
"bucket-${var.name}-${var.env}"